examples/Transports/FIX/over_one_session.py

```diff
@@ -28,6 +29,7 @@
 from testplan.common.utils.context import context
 from testplan.testing.multitest import MultiTest, testsuite, testcase
 from testplan.testing.multitest.driver.fix import FixServer, FixClient
+from testplan.common.utils.sockets.tls import SimpleTLSConfig
 
 CODEC = Codec(spec=FixSpec(SPEC_FILE))
```

Comment:
Independent of current scope, but why do we use this approach here? The Codec from pyfixmsg work without a specification even though it will have issues with groups and encoded tags.

Reply:
I think this is to demonstrate to user about how to provide the fix spec.